include/lsst/afw/image/Mask.h

```diff
@@ -72,6 +72,10 @@ using MaskPlaneDict = std::map<std::string, int>;
  * - `INTRP` This pixel has been interpolated over @note should be called `INTERPOLATED`
  * - `SAT` This pixel is saturated and has bloomed @note should be called `SATURATED`
  * - `SUSPECT` This pixel is untrustworthy, and you may wish to discard any Source containing it
+ * - `NO_DATA` This pixel could not have received flux, e.g. fully vignetted, no available flux from coadd
```

Comment:
```suggestion
 * - `NO_DATA` This pixel received little or no flux, e.g. fully vignetted, no available flux from coadd
```
It's usually useful to start setting this mask plane before the flux level drops all the way to zero.